src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -94,6 +97,40 @@ public BoxSharedLink createSharedLink(BoxSharedLink.Access access, Date unshareD
     }
 
     /**
+     * Adds a new task to this file. The task can have an optional message to include, and a due date
```

Comment:
Please put a full stop at the end of the first line in comments for consistency.